testdata.log

```diff
@@ -8412,23 +8412,17 @@ SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 ) ORDER BY i;
 
 ---- 1349
 SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );
```

Comment:
@cznic  by doing what I mentioned bove the testdata.log entry looks like

```
---- 1349
SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );
┌Iterate all rows of table "t"
└Output field names ["i"]
┌Filter on false
└Output field names ["i"]
```

Is that okay?